### PR TITLE
Add areas represented in Congress endpoint

### LINKED ISSUE #78: States appear in two places
Currently, the front-end and back-end each have their own hard-coded list of the 50 United States + territories. This design may confuse future developers, and may make State bugs/typos harder to find. Opening discussion to decide if creating one master list of the States is worth it, and what our options are.

### PR DESCRIPTION
**Pull request by kevincianfarini**
Closes #78

### CODE REVIEW

**Comment by kevincianfarini** [comment]
This PR only adds support for a simple list of state abbreviations. I figured that would be sufficient for right now. IF it's not, I can revisit and add full names as well. Let me know what you think @jacknugent.

File: backend/src/main/kotlin/org/climatechangemakers/act/plugins/Serialization.kt [hunk 1]
```diff
@@ -5,9 +5,15 @@ import io.ktor.application.install
...
 import kotlinx.serialization.json.Json
+import kotlinx.serialization.modules.serializersModuleOf
+import org.climatechangemakers.act.common.model.RepresentedAreaSerializer
 
 fun Application.configureSerialization() {
   install(ContentNegotiation) {
-    json(Json { prettyPrint = true })
+    val json = Json {
+      prettyPrint = true
+      serializersModule = serializersModuleOf(RepresentedAreaSerializer)
```
Comment (kevincianfarini) [inline backend/src/main/kotlin/org/climatechangemakers/act/plugins/Serialization.kt:15]: I don't like this. Normally serializers in kotlin are statically resolved (and therefore really fast), but this serializer for some reason requires runtime resolution. I asked a question in the kotlin slack, and may follow up with this later.

**Comment by jacknugent** [comment]
> This PR only adds support for a simple list of state abbreviations. I figured that would be sufficient for right now. IF it's not, I can revisit and add full names as well. Let me know what you think @jacknugent.

How about we leave the issue open and maybe address it down the line, time permitting?

**Comment by kevincianfarini** [comment]
Sure thing. Mind approving?

**Review by jacknugent — APPROVED** [review]

**Comment by kevincianfarini** [comment]
@jacknugent I fixed the PR to serialize names and short codes. I'm going to merge and close. 

```js
[
    {
        "shortName": "AK",
        "fullName": "Alaska"
    },
    {
        "shortName": "AL",
        "fullName": "Alabama"
    },
    {
        "shortName": "AS",
        "fullName": "American Samoa"
    },
    {
        "shortName": "AZ",
        "fullName": "Arizona"
    },
    {
        "shortName": "AR",
        "fullName": "Arkansas"
    },
    {
        "shortName": "CA",
        "fullName": "California"
    },
    {
        "shortName": "CO",
        "fullName": "Colorado"
    },
    {
        "shortName": "CT",
        "fullName": "Connecticut"
    },
    {
        "shortName": "DC",
        "fullName": "District Of Columbia"
    },
    {
        "shortName": "DE",
        "fullName": "Delaware"
    },
    {
        "shortName": "FL",
        "fullName": "Florida"
    },
    {
        "shortName": "GA",
        "fullName": "Georgia"
    },
    {
        "shortName": "GU",
        "fullName": "Guam"
    },
    {
        "shortName": "HI",
        "fullName": "Hawaii"
    },
    {
        "shortName": "ID",
        "fullName": "Idaho"
    },
    {
        "shortName": "IL",
        "fullName": "Illinois"
    },
    {
        "shortName": "IN",
        "fullName": "Indiana"
    },
    {
        "shortName": "IA",
        "fullName": "Iowa"
    },
    {
        "shortName": "KS",
        "fullName": "Kansas"
    },
    {
        "shortName": "KY",
        "fullName": "Kentucky"
    },
    {
        "shortName": "LA",
        "fullName": "Louisiana"
    },
    {
        "shortName": "ME",
        "fullName": "Maine"
    },
    {
        "shortName": "MD",
        "fullName": "Maryland"
    },
    {
        "shortName": "MA",
        "fullName": "Massachusetts"
    },
    {
        "shortName": "MI",
        "fullName": "Michigan"
    },
    {
        "shortName": "MN",
        "fullName": "Minnesota"
    },
    {
        "shortName": "MS",
        "fullName": "Mississippi"
    },
    {
        "shortName": "MO",
        "fullName": "Missouri"
    },
    {
        "shortName": "MT",
        "fullName": "Montana"
    },
    {
        "shortName": "NE",
        "fullName": "Nebraska"
    },
    {
        "shortName": "NV",
        "fullName": "Nevada"
    },
    {
        "shortName": "NH",
        "fullName": "New Hampshire"
    },
    {
        "shortName": "NJ",
        "fullName": "New Jersey"
    },
    {
        "shortName": "NM",
        "fullName": "New Mexico"
    },
    {
        "shortName": "NY",
        "fullName": "New York"
    },
    {
        "shortName": "NC",
        "fullName": "North Carolina"
    },
    {
        "shortName": "ND",
        "fullName": "North Dakota"
    },
    {
        "shortName": "MP",
        "fullName": "Northern Mariana Islands"
    },
    {
        "shortName": "OH",
        "fullName": "Ohio"
    },
    {
        "shortName": "OK",
        "fullName": "Oklahoma"
    },
    {
        "shortName": "OR",
        "fullName": "Oregon"
    },
    {
        "shortName": "PA",
        "fullName": "Pennsylvania"
    },
    {
        "shortName": "PR",
        "fullName": "Puerto Rico"
    },
    {
        "shortName": "RI",
        "fullName": "Rhode Island"
    },
    {
        "shortName": "SC",
        "fullName": "South Carolina"
    },
    {
        "shortName": "SD",
        "fullName": "South Dakota"
    },
    {
        "shortName": "TN",
        "fullName": "Tennessee"
    },
    {
        "shortName": "TX",
        "fullName": "Texas"
    },
    {
        "shortName": "UT",
        "fullName": "Utah"
    },
    {
        "shortName": "VI",
        "fullName": "Virgin Islands"
    },
    {
        "shortName": "VT",
        "fullName": "Vermont"
    },
    {
        "shortName": "VA",
        "fullName": "Virginia"
    },
    {
        "shortName": "WA",
        "fullName": "Washington"
    },
    {
        "shortName": "WV",
        "fullName": "West Virginia"
    },
    {
        "shortName": "WI",
        "fullName": "Wisconsin"
    },
    {
        "shortName": "WY",
        "fullName": "Wyoming"
    }
]
```

**Pull request merged**